src/server/controllers/createaccountController.js

```diff
@@ -190,9 +190,9 @@ function checkUsernameAvailable(req, res) {
     let allowed = true;
     let reason = '';
 
-    if (doesMemberExist(usernameLowercase)) { allowed = false; reason = 'That username is taken'; }
-    if (checkProfanity(usernameLowercase)) { allowed = false; reason = 'That username contains a word that is not allowed'; }
-    if (reservedUsernames.includes(usernameLowercase)) { allowed = false; reason = 'That username is taken'; } // Code for reserved (but the users don't know that!)
+    if (doesMemberExist(usernameLowercase)) { allowed = false; reason = "ws-username_taken"; }
+    if (checkProfanity(usernameLowercase)) { allowed = false; reason = "ws-username_bad_word"; }
+    if (reservedUsernames.includes(usernameLowercase)) { allowed = false; reason = "ws-username_taken"; } // Code for reserved (but the users don't know that!)
```

Comment:
Maybe we should give this one it's own code, in case in the future we do want to tell them if it's reserved, vs taken. I mean otherwise how would these big reserved names know they could claim their name?

Reply:
Hm, I thought I remembered a conversation where the current version was settled upon and I didn't change any text when localizing, but it's fine by me anyways to change it. I suppose there is a nonzero chance and we should add new localizations sooner rather than later.

Reply:
Done!